app/assets/stylesheets/base/settings_colours.scss

```diff
@@ -66,3 +67,5 @@ $progress-background:     $light-grey-orange;
 $progress-edits:          $bright-yellow;
 $progress-awaiting:       $bright-orange;
 $progress-completed:      $mid-green;
+
+$transcript-background:   lighten($light-grey-orange, .2);
```

Comment:
Using `lighten` and similar quickly increases the total number of colors in your palette - use sparingly.